scripts/format_all.sh

```diff
@@ -4,7 +4,9 @@ set -e
 
 cd "$(dirname "$0")/.."
 
-for cmd in clang-format find grep python3 sed xargs; do
+CLANG_FORMAT_BIN="${CLANG_FORMAT_BIN:-clang-format}"
```

Comment:
If CC/CXX is clang, you can also try to find its corresponding clang-format.

Reply:
We may not set CC/CXX when calling the formatter